cmd/utils/flags.go

```diff
@@ -723,6 +723,12 @@ var (
 		Usage: "Port for sentinel",
 		Value: 7777,
 	}
+
+	DisableIPV6 = cli.BoolFlag{
+		Name:  "disable.ipv6",
```

Comment:
Name of the flag needs to mention `downloader.` because it is setting for that component only